### PR TITLE
Portfolio Project

### PR DESCRIPTION
**Pull request by danigalindo10**
# Portfolio 
In this project I:
- Added an Html and CSS file to create the portfolio.
- Used the Figma template to use it in my project.
- Configured stick to the Figma design as much as possible.

### CODE REVIEW

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @danigalindo10 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project that needs to complete all linters configuration. You are remaining with the[ following steps ](https://github.com/microverseinc/linters-config/tree/master/html-css#:~:text=Webhint,a%20clean%20code!)to correctly set up your linters. Please, kindly start with the first command of initialization and read all steps. I highlighted what you will do.

**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏**

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please the hamburger icon should be inside a ``nav tag`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ optional] you can use your name in place of `my logo`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
+    <div class="section-title"><h1>I'm Daniel <br>Glad to see <br>you!</h1></div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] i'm daniel should be in an `h1 tag` while glad to see you is an `h2.` please fix that

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="section-content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please put this outline text inside a `p tag`.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  </section>
+  <footer>
+    <div class="footer-content">LET'S CONNECT</div>
+    <div class="main-footer-network-content">
+      <div class="footer-network twitter"><img src="Images/twitter.svg" alt="twitter"></div>
+      <div class="footer-network linkedin"><img src="Images/linkedin.svg" alt="linkedin"> </div>
+      <div class="footer-network M"> <img src="Images/m.svg" alt="M"> </div>
+      <div class="footer-network github"> <img src="Images/github.svg" alt="github"> </div>
+      <div class="footer-network hand"> <img src="Images/hand.svg" alt="hand"> </div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please ensure that your social icons are inside a `ul tag` and each of them in a `li tag`.
- [ ] Add link place holders to each icon ..`<a href="/"></a>` .you can check how to add that [here](https://www.w3schools.com/html/html_links.asp)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
```
Comment (tufoinnkuo10) [inline index.html:21]: - [ ] please try to put everything in your headline inside the section container.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="section-title"><h1>I'm Daniel <br>Glad to see <br>you!</h1></div>
+    <div class="section-content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+  </section>
+  <footer>
+    <div class="footer-content">LET'S CONNECT</div>
+    <div class="main-footer-network-content">
+      <div class="footer-network twitter"><img src="Images/twitter.svg" alt="twitter"></div>
+      <div class="footer-network linkedin"><img src="Images/linkedin.svg" alt="linkedin"> </div>
+      <div class="footer-network M"> <img src="Images/m.svg" alt="M"> </div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please try to make your project look almost exactly like that of the figma. that is you follow the fonts, colors, margins etc. align your lest connect text and icons well. 
- your design 👇🏾 
Screenshot (46)_LI
- figma design [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👇🏾 
Screenshot (39)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @danigalindo10 

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

**To Highlight**
✅ You used the correct Github flow
✅ Good commit messages
✅ linter checks validated
✅ Good pull request summary
✅ Correct Readme template

Required Changes ♻️
Check the comments under the review

**Optional suggestions**
- [ optional ] please try to commit more often and have reasonable commit messages
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 6 above)
Comment (danigalindo10) [reply index.html:21]: Hello @tufoinnkuo10, did you mean to do this? Put my header inside the section container?

```
<section class="section-text">
    <header class="header">
      <div class="header-content">
        <div class="logo">
          <a href="#">Daniel</a>
        </div>
          <nav>
            <img class="hamburger" src="images/menu.svg" alt="menu">
          </nav>
      </div>
    </header>
    <div class="section-title">
      <h1>I'm Daniel</h1> 
        <h2>Glad to see you!</h2>
    </div>
    <div class="section-content">
      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
    </div>
</section>
```

**Comment by danigalindo10** [comment]
# Portfolio changes
I have just done all the changes you require me to do @tufoinnkuo10. Thank you for your feedback.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">Daniel</a>
+      </div>
+        <nav>
+          <img class="hamburger" src="images/menu.svg" alt="menu">
```
Comment (Mre55) [inline index.html]: Kindly, fix the `src` of the image currently it's referencing the wrong directory and the result showing on the page is like this 👇 

image

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @danigalindo10,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by danigalindo10** [comment]
# Fixed src path
Hello @Mre55, I've just changed the src path for the project.

**Review by smunozmo — APPROVED** [review]
Hi @danigalindo10, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**